galite-core/src/main/kotlin/org/kopi/galite/form/VMonthField.kt

```diff
@@ -32,7 +32,7 @@ import org.kopi.galite.visual.VException
 import org.kopi.galite.visual.VlibProperties
 import java.sql.Date
 
-class VMonthField(val bufferSize: Int) : VField(7, 1) {
+class VMonthField(val bufferSize: Int) : VField(30, 1) {
```

Comment:
this cause regression on other UI implementation.
Instead of changing in the model you should enhance the UI implementation